packages/backend-tasks/src/tasks/TaskWorker.ts

```diff
@@ -51,6 +51,10 @@ export class TaskWorker {
       `Task worker starting: ${this.taskId}, ${JSON.stringify(settings)}`,
     );
 
+    const cadence = Duration.fromISO(settings.cadence);
```

Comment:
I may be misremembering, but can't the cadence be a cron string too? (Not at computer now)

Reply:
Yep, updated!